frontend/src/metabase/query_builder/containers/QueryBuilder.jsx

```diff
@@ -160,8 +160,8 @@ const mapStateToProps = (state, props) => {
 const mapDispatchToProps = {
   ...actions,
   onChangeLocation: push,
-  createBookmark: id => Bookmark.actions.create({ id }),
-  deleteBookmark: id => Bookmark.actions.delete({ id }),
+  createBookmark: id => Bookmark.actions.create({ id: `card-${id}` }),
```

Comment:
After the fix within `entities.js` this could be changed to `Bookmark.actions.create({ id, entity: "card" })`